application/controllers/AdminController.php

```diff
@@ -498,6 +498,14 @@ public function getAdminFooter($url, $explanation, $return = false)
             $aData['versiontitle'] = gT('Version');
         }
 
+        $oEvent = new PluginEvent('beforeFooterRender');
+        App()->getPluginManager()->dispatchEvent($oEvent);
+        $data = $oEvent->get('data');
```

Comment:
If multiple plugins are using the same event, you might need to append the data rather than just getting it. Or we don't care about that.